### PR TITLE
Update networks.mdx

### PR DESCRIPTION
**Pull request by antrixy**
Added Antrix Validators end points.

### CODE REVIEW

File: docs/develop/api/networks.mdx [hunk 1]
```diff
@@ -63,6 +63,10 @@ If you are searching for the protobuf interfaces, head over [here](https://buf.b
 | `https://rpc-evmos.mms.team:443`              | `Tendermint` `RPC`     | [MMS](https://mms.team/) | Pruned  |
 | `https://jsonrpc-evmos.mms.team:443`          | `Ethereum` `JSON-RPC`  | [MMS](https://mms.team/) | Pruned  |
 | `https://web3endpoints.com/evmos-mainnet`     | `Ethereum` `JSON-RPC`  | [BlockSpaces](https://www.blockspaces.com/web3-infrastructure) | Pruned |
+| `https://evmos-rest.antrixy.org`              | `Cosmos` `REST`        | [Antrix Validators]                             | Pruned  |
```
Comment (GAtom22) [inline docs/develop/api/networks.mdx:66]: Please, add a link to your site or remove the brackets
```suggestion
| `https://evmos-rest.antrixy.org`              | `Cosmos` `REST`        | Antrix Validators                             | Pruned  |
```

File: docs/develop/api/networks.mdx [hunk 2]
```diff
@@ -89,6 +93,9 @@ If you are searching for the protobuf interfaces, head over [here](https://buf.b
 | `https://evmos-testnet-lcd.qubelabs.io`      | `Cosmos` `REST`        | [Qubelabs](https://qubelabs.io/)        | Pruned  |
 | `https://evmos-testnet-grpc.qubelabs.io`     | `Cosmos` `gRPC`        | [Qubelabs](https://qubelabs.io/)        | Pruned  |
 | `https://evmos-testnet-json.qubelabs.io`     | `Ethereum` `JSON-RPC`  | [Qubelabs](https://qubelabs.io/)        | Pruned  |
+| `https://evmos-trpc.antrixy.org`             | `Tendermint` `RPC`     | [Antrix Validators]                     | Pruned  |
```
Comment (GAtom22) [inline docs/develop/api/networks.mdx:96]: ditto

**Review by GAtom22 — CHANGES_REQUESTED** [review]
Thanks for the contribution @antrixy!
Left one comment to address

File: docs/develop/api/networks.mdx (hunk 2 above)
Comment (antrixy) [reply docs/develop/api/networks.mdx]: Done

File: docs/develop/api/networks.mdx [hunk 3]
```diff
@@ -63,6 +63,10 @@ If you are searching for the protobuf interfaces, head over [here](https://buf.b
 | `https://rpc-evmos.mms.team:443`              | `Tendermint` `RPC`     | [MMS](https://mms.team/) | Pruned  |
 | `https://jsonrpc-evmos.mms.team:443`          | `Ethereum` `JSON-RPC`  | [MMS](https://mms.team/) | Pruned  |
 | `https://web3endpoints.com/evmos-mainnet`     | `Ethereum` `JSON-RPC`  | [BlockSpaces](https://www.blockspaces.com/web3-infrastructure) | Pruned |
+| `https://evmos-rest.antrixy.org`              | `Cosmos` `REST`        | Antrix Validators                               | Pruned  |
+| `https://evmos-grpc.antrixy.org`              | `Cosmos` `gRPC`        | Antrix Validators                               | Pruned  |
+| `https://evmos-rpc.antrixy.org`               | `Tendermint` `RPC`     | Antrix Validators                               | Pruned  |
+| `https://evmos-json.antrixy.org`              | `Ethereum` `JSON-RPC`  | Antrix Validators                               | Pruned  |
```
Comment (MalteHerrmann) [inline docs/develop/api/networks.mdx:69]: These seem to be non-functional 🤔

**Review by MalteHerrmann — CHANGES_REQUESTED** [review]
Hey there! Love to see new RPC providers on Evmos, however the mainnet endpoints seem to not work ATM. Can you check? 🙏

File: docs/develop/api/networks.mdx (hunk 3 above)
Comment (antrixy) [reply docs/develop/api/networks.mdx:69]: It is functional. Please check again.

Comment (MalteHerrmann) [reply docs/develop/api/networks.mdx:69]: They do now 👍 they didn't work when I tried them back on Nov. 2nd

File: docs/develop/api/networks.mdx [hunk 4]
```diff
@@ -89,6 +93,9 @@ If you are searching for the protobuf interfaces, head over [here](https://buf.b
 | `https://evmos-testnet-lcd.qubelabs.io`      | `Cosmos` `REST`        | [Qubelabs](https://qubelabs.io/)        | Pruned  |
 | `https://evmos-testnet-grpc.qubelabs.io`     | `Cosmos` `gRPC`        | [Qubelabs](https://qubelabs.io/)        | Pruned  |
 | `https://evmos-testnet-json.qubelabs.io`     | `Ethereum` `JSON-RPC`  | [Qubelabs](https://qubelabs.io/)        | Pruned  |
+| `https://evmos-trpc.antrixy.org`             | `Tendermint` `RPC`     | Antrix Validators                       | Pruned  |
+| `https://evmos-trest.antrixy.org`            | `Cosmos` `REST`        | Antrix Validators                       | Pruned  |
+| `https://evmos-tgrpc.antrixy.org`            | `Cosmos` `gRPC`        | Antrix Validators                       | Pruned  |
```
Comment (MalteHerrmann) [inline docs/develop/api/networks.mdx:98]: @antrixy do you guys have a Ethereum JSON RPC endpoint for testnet too?

**Review by MalteHerrmann — APPROVED** [review]
LGTM, only one question on the Ethereum endpoint for testnet

File: docs/develop/api/networks.mdx (hunk 4 above)
Comment (MalteHerrmann) [reply docs/develop/api/networks.mdx:88]: ok no problem, just checking! 👍 will merge now

**Pull request merged**